src/Tools/dotnet-counters/Program.cs

```diff
@@ -97,14 +97,31 @@ internal class Program
                 name: "list",
                 description: "Display a list of counter names and descriptions, grouped by provider.")
             {
-                Handler = CommandHandler.Create<IConsole>(List)
+                CommandHandler.Create<IConsole, string>(List),
+                RuntimeVersionOption()
             };
 
-        public static int List(IConsole console)
+        private static Option RuntimeVersionOption() =>
+            new Option(
+                aliases: new[] { "-r", "--runtime-version" },
+                description: "Version of runtime. Supported runtime version: 3.0, 3.1, 5.0") 
```

Comment:
With the time delta between 5.0 and 6.0, I could see counters being added in the interim via servicing releases.  I imagine these values may need to differentiate by the build number as well.  Not right now, since we don't have any counters that are "in between" like that currently, but it is something to keep in mind.

Reply:
I've never added any counters in servicing releases in the past, and I don't think I will be adding counters in servicing releases since they're treated as features and not bug fixes.